### DISCUSSION TITLE
在插件内部，再次导入二级目录的插件路径问题

### DISCUSSION THREAD

**Discussion by s52047qwas** [nonebot/discussions#226] [问答]
当前插件目录为：
一级目录
   __init__.py
   二级目录
      __init__.py

当为load_all_plugins导入使用时：
load_all_plugins(
        [
            '一级目录.二级目录',
        ],
    )

会遇到一下问题：
1、当插件为python的install导入时，是可以正常使用的
2、但是当插件为在本地的plugins的目录时，需要加入src.plugins才可以正常导入
'src.plugins.一级目录.二级目录'

这个有其他的快捷的方法可以使用吗

**Comment by yanyongyu** [comment]
本地插件没有前缀模块名python哪知道你插件在哪个文件夹，肯定得要完整的路径
